src/core/public/core_system.ts

```diff
@@ -242,6 +248,7 @@ export class CoreSystem {
         overlays,
         savedObjects,
         uiSettings,
+        workspaces,
```

Comment:
@wanglam I was trying to align the convention with the existing names.

Reply:
Oh, I get your point. You're right, I think I'd like change it to `workspace`, let me see how many places affected